src/main/java/com/adelin/project/student_service/controller/Student_Controller.java

```diff
@@ -0,0 +1,55 @@
+package com.adelin.project.student_service.controller;
+
+import com.adelin.project.student_service.entity.Student;
+import com.adelin.project.student_service.service.Student_Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+public class Student_Controller {
```

Comment:
pentru controller facade si service iti trebe o intefata numita StudentController in cazul asta si asta care o implementeaza o sa o cheme StudentControllerImpl